.github/workflows/ci.yml

```diff
@@ -5,16 +5,16 @@ name: CI
 permissions: read-all
 on:
   pull_request:
-    branches:
-      - main
+#    branches:
```

Comment:
?

Reply:
I'm using this to be able run acc tests on PR on my fork, can revert it